### DISCUSSION TITLE
Chia Never Fully Synced

### DISCUSSION THREAD

**Discussion by endjinn35** [Chia-Network/chia-blockchain#5735] [Support]
I have been trying to get my farming computer to sync for the past week. It took 5 days to reach 311,000/314,000 and the goal just kept increasing before it ever got there. I have tried Everything i can find on help forums.
  port forwarding to 8444, disabling and re-enabling uPnP, manually connecting to peers, digging through my bug log for answers, etc. I am at a complete loss for ideas . I've even tried abandoning my wallet on the computer, uninstalling chia and deleting the files, and starting  everything from the ground up. now it's taken 10 minutes to get 600/290,000 sync so the problem seems to persist. Thank you for your time and help. If there's any advice you can give or things you could perform, please let me know and thank you again.
 *on every wallet I start my blocks list their created time at 2 days ago or yesterday which makes me think the clock is out of synch but I haven't found a lot of direct information on how to address this issue. I was hoping starting a new wallet from the ground up would fix this.
   any advice or help would be greatly appreciated thank you for your time and happy Friday!

**Comment by BrandtH22** [comment]
@endjinn35 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
